tsl/src/fdw/data_node_scan_plan.c

```diff
@@ -201,11 +201,19 @@ adjust_data_node_rel_attrs(PlannerInfo *root, RelOptInfo *data_node_rel, RelOptI
 								make_restrictinfo_compat(root,
 														 (Expr *) onecq,
 														 rinfo->is_pushed_down,
+#if PG16_LT
+														 false,
+														 false,
+#else
+														 rinfo->has_clone,
+														 rinfo->is_clone,
+#endif
 														 rinfo->outerjoin_delayed,
```

Comment:
It is, in general, not a good idea to have preprocessor directives inside the parameter list of a preprocessor macro. Weird issues can result. See https://wiki.sei.cmu.edu/confluence/display/c/PRE32-C.+Do+not+use+preprocessor+directives+in+invocations+of+function-like+macros